docs/managing-compute-resources.asciidoc

```diff
@@ -25,7 +25,14 @@ For Elasticsearch objects, make sure to consider the heap size when you set reso
 ----
 spec:
   nodeSets:
-  - podTemplate:
+  - name: elasticsearch
```

Comment:
I think we can reuse the [quickstart naming scheme](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html#k8s-deploy-elasticsearch) here and name this nodeSet `default`.